### ISSUE TITLE
Copy-paste not working

### ISSUE THREAD

**Issue by dflat1** [jlesage/docker-firefox#110]
Hi,

After installing the latest version, and running the browser in Google Chrome / Firefox, copying text from the docker firefox http webui did not work.  Also, I couldn't paste text from the host computer into the docker firefox http webui.

Is it possible to fix the copy-paste functionality?

**Comment by BrunoX66** [comment]
I can confirm the issue. Clipboard functionality is not working anymore in the latest release, at least not through browser.

**Comment by jlesage** [comment]
I'm able to reproduce.  Copy/paste from container to host does seem to work, but not the other way.

**Comment by trancemind** [comment]
Same problem here.. Forced to rollback to older version v1.17.1, because a clipboard feature is critical.
@jlesage is any progress in the issue resolving or maybe there is some recommendations of how to fix this, e.g. by changing X11vnc startup commands? Thank you.

**Comment by barucoh** [comment]
I'm experiencing the same issue, is there a fix in the works?

**Comment by jlesage** [comment]
I'm still investigating, I'm not sure yet what is causing the problem.  The same base image is working fine with other applications, so it looks like Firefox itself might have something to do with the problem.

**Comment by barucoh** [comment]
Hi @jlesage, sorry to bring this up again but are there any news about this?

**Comment by Un4gi** [comment]
Having the same issue - I'm able to copy/paste from the container to the host but not the other way around. I see that the issue is being investigated, fingers crossed!

**Comment by typkrft** [comment]
I wasn't able to figure out much about what's going on either. I did update Firefox in one of the older images that was working and it seemed to break it, but I'm not sure if this was a firefox issue or maybe some dependency that did this. I was thinking about rolling my own based on an ubuntu image, but additional resources needed made it not an attractive option.

**Comment by tchirou** [comment]
Hello
Would something be in the works to fix this ?
Many thanks !

**Comment by jlesage** [comment]
Yes, hopefully this will be fixed in the next version, using the new base image.

**Comment by CaTeNdrE** [comment]
Hi  @jlesage any ETA on this fix?  Ty!

**Comment by proteus077** [comment]
Following this issue as I'm also very interested when a fix is released
